app/scripts/models/reliers/oauth.js

```diff
@@ -157,6 +157,12 @@ define(function (require, exports, module) {
       // https://github.com/mozilla/fxa-oauth-server/blob/master/docs/api.md#post-v1authorization
       this.importSearchParamsUsingSchema(
           SIGNIN_SIGNUP_QUERY_PARAM_SCHEMA, OAuthErrors);
+
+      // OAuth reliers are not allowed to specify both a client_id and a service
```

Comment:
same as https://github.com/mozilla/fxa-content-server/pull/5050/files#r116279959 

` OAuth reliers are not allowed to specify both a client_id and a service` somehow should be ` OAuth reliers are not allowed to specify service`